common/src/main/java/jp/co/soramitsu/common/compose/component/Toolbar.kt

```diff
@@ -0,0 +1,178 @@
+package jp.co.soramitsu.common.compose.component
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Arrangement.spacedBy
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.IconButton
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Alignment.Companion.CenterHorizontally
+import androidx.compose.ui.Alignment.Companion.CenterVertically
+import androidx.compose.ui.Alignment.Companion.End
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import jp.co.soramitsu.common.R
+import jp.co.soramitsu.common.compose.theme.backgroundBlurColor
+import jp.co.soramitsu.common.compose.theme.white
+import jp.co.soramitsu.common.compose.theme.customTypography
+import jp.co.soramitsu.common.compose.theme.FearlessTheme
+import jp.co.soramitsu.common.compose.theme.colorAccent
+
+
+data class MainToolbarViewState(
+    val title: String,
+    val selectedChainName: String,
+    val selectedChainId: String,
+    val selectedChainStatusColor: Color
+)
```

Comment:
`val selectedChainName: String,
    val selectedChainId: String,
    val selectedChainStatusColor: Color`

Extract these val's into separated ChainSelectorViewState and add it to the MainToolbarViewState as a constructor property

Reply:
Fixed